README.md

```diff
@@ -4,28 +4,11 @@
 [![crates.io](https://img.shields.io/crates/v/log4rs.svg)](https://crates.io/crates/log4rs)
 [![License: MIT OR Apache-2.0](https://img.shields.io/crates/l/clippy.svg)](#license)
 ![CI](https://github.com/estk/log4rs/workflows/CI/badge.svg)
-[![Minimum rustc version](https://img.shields.io/badge/rustc-1.69+-green.svg)](https://github.com/estk/log4rs#rust-version-requirements)
+[![Minimum rustc version](https://img.shields.io/badge/rustc-1.70+-green.svg)](https://github.com/estk/log4rs#rust-version-requirements)
 
 log4rs is a highly configurable logging framework modeled after Java's Logback
 and log4j libraries.
 
-## Warning
-
-If you are using the file rotation in your configuration there is a known
-substantial performance issue so listen up! By default the `gzip` feature
-is enabled and when rolling files it will zip log archives automatically.
-This is a problem when the log archives are large as the zip happens in the
-main thread and will halt the process while the zip is completed. Be advised
-that the `gzip` feature will be removed from default features as of `1.0`.
-
-The methods to mitigate this are as follows.
-
-1. Use the `background_rotation` feature which spawns an os thread to do the compression.
-1. Disable the `gzip` feature with `--no-default-features`.
-1. Ensure the archives are small enough that the compression time is acceptable.
-
-For more information see the PR that added [`background_rotation`](https://github.com/estk/log4rs/pull/117).
-
```

Comment:
why rm?

Reply:
so, i just pulled the thread and although gzip is no-longer part of the default features, I still think a warning regarding the use of gzip is important to be featured in the readme... perhaps not as prominently. Also, this change should be pulled out into a separate PR.